doc/Command-Reference.md

```diff
@@ -3129,7 +3132,23 @@ This command displays the running configuration of the snmp module.
 
 Go Back To [Beginning of the document](#SONiC-COMMAND-LINE-INTERFACE-GUIDE) or [Beginning of this section](#Startup--Running-Configuration)
 
+# SNMP Community  Configuration
 
+## snmp-community config command
+
+  This command is used to set SNMP community string or add more string to snmp_rocommunities.
```

Comment:
>set SNMP community string or add [](start = 26, length = 32)

The description is not clear.
1. How to replace with a new one?
2. How to replace with a new list?
3. How to append a new one with existing one or list?
4. Optional, how to append a new list with existing one or list?

Reply:
Currently, code is designed in such a fashion so that it will first check the file **_/etc/sonic/snmp.yml_**. If there's single community string like `snmp_rocommunity: public` and the intention is to add another community string named  "new_public", it will replace the key `snmp_rocommunity` with `snmp_rocommunities` and both the old and new community strings will be added to the new key i.e. `snmp_rocommunities`.

**For example**

before
```
snmp_rocommunity:
- public
```
After
```
snmp_rocommunities:
- public
- new_public
```

But if `snmp_rocommunities` is already there in **_/etc/sonic/snmp.yml_**, it will just append the  new community string to the existing key 


**For example**

before
```
snmp_rocommunities:
- public
- new_public

```
After
```
snmp_rocommunities:
- public
- new_public
- new_public_2
```

Reply:
I got your idea, then I suggest change the comment to align with your implementation.
```
  This command is used to add SNMP community string to snmp_rocommunities
```